lib/blockchain.rb

```diff
@@ -19,8 +19,9 @@ def create_transaction(data, time = Time.new)
     unverified_transactions.push(transaction)
   end
 
-  def create_block
-    block = Block.new(unverified_transactions, blocks.last.hash)
+  def create_block(block_class = Block)
```

Comment:
What's the reason for the block_class change?

Reply:
Helps me mock, dependency injection

Reply:
Nice